### ISSUE TITLE
The microsoft teams webhook has a url configured which is pointing to the wrong link in prefect UI

### ISSUE THREAD

**Issue by prshnt1314** [PrefectHQ/prefect#7206]
### First check

- [X] I added a descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Prefect documentation for this issue.
- [X] I checked that this issue is related to Prefect and not one of its dependencies.

### Bug summary

The microsoft teams webhook has a url configured which is pointing to the wrong link in prefect UI

### Reproduction

```python3
1. add notifications using the notification feature for MSTeams
2. trigger a run and invoke it based on flow completion
3. wait for the message to appear in teams channel
4. observe the link shared in the channel is pointing to a 404 URL


expected: http://localhost:4200/flow-run/e09b8fac-8587-492e-8b97-acd2ff5eabd5
actual http://localhost:4200/flow-runs/flow-run/e09b8fac-8587-492e-8b97-acd2ff5eabd5
```

### Versions

```Text
Version:             2.6.1
API version:         0.8.2
Python version:      3.8.15
Git commit:          d9dd4443
Built:               Fri, Oct 14, 2022 2:01 PM
OS/Arch:             linux/x86_64
Profile:             default
Server type:         hosted
```

**Comment by rpeden** [comment]
I believe this is addressed by #7153 and should be fixed in the next release.

**Comment by prshnt1314** [comment]
Thanks @rpeden
